src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -3462,37 +3449,30 @@ void SimAirLoopComponent(EnergyPlusData &state,
         ManageOutsideAirSystem(state, CompName, FirstHVACIteration, AirLoopNum, CompIndex);
         // Fan Types for the air sys simulation
     } break;
-    case CompType::Fan_Simple_CV: { // 'Fan:ConstantVolume'
-        Fans::SimulateFanComponents(state, CompName, FirstHVACIteration, CompIndex);
-    } break;
-    case CompType::Fan_Simple_VAV: { // 'Fan:VariableVolume'
-        Fans::SimulateFanComponents(state, CompName, FirstHVACIteration, CompIndex);
+    case CompType::Fan_Simple_CV:  // 'Fan:ConstantVolume'
+    case CompType::Fan_Simple_VAV: // 'Fan:VariableVolume'
+    case CompType::Fan_ComponentModel: { // 'Fan:ComponentModel'
+        if (CompIndex == 0) {
```

Comment:
This one had to be guarded.

Reply:
So now this check for CompIndex == 0 is no longer needed if the PrimaryAirSystems(AirLoopNum).Branch(BranchNum).Comp(CompNum).CompIndex is set up early.